### DISCUSSION TITLE
ROS2 - Foxy

### DISCUSSION THREAD

**Discussion by samuk** [Roger-random/Sawppy_Rover#32] [General]
I spotted this ROS2 software for Sawppy: https://github.com/mgonzs13/ros2_rover/#readme 
It seems to be a good start, it provides teleoperation and subscribes to /cmd_vel.

I'm hoping to use this deep learning model https://github.com/ANI717/ANI717_Robotics#design-diagram to publish to /cmd_vel.

**Comment by knickels** [comment]
Any update on this?  I'm about ready to start putting ros2 on my rover.
